src/radians.rs

```diff
@@ -34,6 +34,11 @@ impl Radians {
     pub fn value(self) -> f64 {
         self.value
     }
+
+    /// Convert degrees to radians
+    pub fn from_degrees(degrees: f64) -> Result<Self, RadiansError> {
+        return Radians::try_new(degrees / 360.0 * 2.0 * PI);
```

Comment:
How about
```
const MAX_DEGREES: f64 = 360.0;
const MAX_RADIANS: f64 = 2.0 * PI;
```
?

Reply:
Remove that `return`

Reply:
Resolved in  ae71de9b427dc409ad78b6f205b4e5ff28332e40